### PR TITLE
Feat: use fixed seed, invert order between ground truth and prediction & increase max number of logged images

### PR DESCRIPTION
**Pull request by borisdayma**
This PR proposes 3 changes:

- a fixed seed is used in fastai callback to ensure the same images are always selected at each epoch. This is nice to visualize the evolution of predictions on a same image during training -> see https://app.wandb.ai/borisd13/DeOldify/runs/pff2osv4

- invert order between ground truth and predictions: this allows for nicer displays -> see https://app.wandb.ai/borisd13/DeOldify/runs/x6g016c4

- increase max number of logged images to 108. This is to make it compatible with the default number of samples in keras & fastai callbacks (36) since we want to log 3 images per sample (input, prediction & ground truth)

Let me know what you think? I could also allow to define the `seed` value but think the default should be to a fixed value such as `12345` as it will also give the possibility to compare predictions between runs.

### CODE REVIEW

File: wandb/fastai/__init__.py [hunk 1]
```diff
@@ -92,6 +92,7 @@ def __init__(self,
         # Select items for sample predictions to see evolution along training
         self.validation_data = validation_data
         if input_type and not self.validation_data:
+            random.seed(12345)  # Ensures repeatability of selected samples
```
Comment (vanpelt) [inline wandb/fastai/__init__.py]: I'm wary to call this in the user code as it will seed the random generator for the rest of the script.  Is there a way to do a "scoped" seed, i.e. `with random.seed(12345)` or manually introspect and store the seed before and reset the global seed to it after?

Comment (borisdayma) [reply wandb/fastai/__init__.py]: Great point! It seems it's possible to do scoped seed: https://stackoverflow.com/a/12369081/3474490
Let me do a few tests to ensure this will work.

Comment (borisdayma) [reply wandb/fastai/__init__.py]: This seems to work great!
```
>>> import random
>>> myRandom = random.Random(12345)
>>> random.seed(1)
>>> random.random()
0.13436424411240122
>>> myRandom.random()
0.41661987254534116
```
Let's change the order
```
>>> random.seed(1)
>>> myRandom = random.Random(12345)
>>> myRandom.random()
0.41661987254534116
>>> random.random()
0.13436424411240122
```

Comment (borisdayma) [reply wandb/fastai/__init__.py]: Do you want to leave the option to change the seed (if we don't like the selected examples)?

Comment (vanpelt) [reply wandb/fastai/__init__.py]: Sure, let's make it an argument that defaults to "12345" and if it's None we don't seed the generator.

Comment (borisdayma) [reply wandb/fastai/__init__.py]: Sounds good. Actually the default when creating a Random instance is to pass `None`, which uses the current system time.
